lib/mondrian_redis_segment_cache/cache.rb

```diff
@@ -93,7 +94,6 @@ def put(segment_header, segment_body)
         end
       end
 
-      publish_created_to_listeners(header_base64)
```

Comment:
@ryanbjones yes, when this was running it was seemingly getting stuck at this call which leads me to believe that it can't be done during the `PUT`